packages/experimental/NativeDatePicker/src/NativeDatePicker.tsx

```diff
@@ -1,2 +1,11 @@
-export const NativeDatePicker = null;
-console.warn('DatePicker not supported');
+export const NativeDatePicker = {
```

Comment:
You nshould make a .android version of this file and leave the NativeDatePicker.tsx version as is. Otherwise the NativeDatePicker will not error on other unsupported platforms like macOS.